_analyzers/token-filters/common_gram.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: default
+title: common_grams
+parent: Token filters
+nav_order: 60
+---
+<!-- vale off -->
+# Common_grams token filter
+<!-- vale on -->
+The `common_grams` token filter in OpenSearch improves search relevance by keeping commonly occurring phrases (common grams) in the text. This is useful when dealing with languages or datasets where certain word combinations frequently occur and can impact the search relevance if treated as separate tokens.
+
+Using this token filter improves search relevance by keeping common phrases intact, it can help in matching queries more accurately, particularly for frequent word combinations. It also improves search precision by reducing the number of irrelevant matches.
+
+Using this filter requires careful selection and maintenance of the list of common words
+{: .warning}
+
+## Parameters
+
+`common_grams` token filter can be configured with several parameters to control its behavior.
+
+Parameter | Description | Example
+`common_words` | A list of words that should be considered as common words. These words will be used to form common grams. (Required) | ["the", "and", "of"]
```

Comment:
NOTE:When the common_words parameter is given an empty list, the common_grams token filter essentially becomes a pass-through filter, meaning it doesn't modify the input tokens at all.